lib/matplotlib/rcsetup.py

```diff
@@ -615,28 +500,6 @@ def validate_markevery(s):
 
 validate_markeverylist = _listify_validator(validate_markevery)
 
-validate_legend_loc = ValidateInStrings(
-    'legend_loc',
-    ['best',
-     'upper right',
-     'upper left',
-     'lower left',
-     'lower right',
-     'right',
-     'center left',
-     'center right',
-     'lower center',
-     'upper center',
-     'center'], ignorecase=True, _deprecated_since="3.3")
-
-validate_svg_fonttype = ValidateInStrings(
-    'svg.fonttype', ['none', 'path'], _deprecated_since="3.3")
-
-
-@_api.deprecated("3.3")
-def validate_hinting(s):
-    return _validate_hinting(s)
```

Comment:
_validate_hinting (below) can be inlined to the main dict.

Reply:
Yep, fixed.